lib/elixir_analyzer/test_suite/log_level.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ElixirAnalyzer.TestSuite.LogLevel do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Log Level
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+
+  assert_call "to_label uses cond/1" do
+    type :actionable
+    comment ElixirAnalyzer.Constants.log_level_use_cond()
+    calling_fn module: LogLevel, name: :to_label
+    called_fn name: :cond
+  end
+
+  assert_call "alert_recipient uses cond/1" do
+    type :actionable
```

Comment:
Let's make both comments `:essential` because they are about fulfilling the core purpose of the exercise as opposed to just general improvements.

Reply:
@Br1ght0ne I see you opening new PRs so I just wanted to do another ping on this one that has been open for a week about this one change 🙂